iOS/PartySample/app/src/SimpleClientImpl.cpp

```diff
@@ -420,10 +422,17 @@ SimpleClientImpl::GetPlayerState() {
     m_messageHandler->OnPlayerStatusUpdateEnd();
 }
 
+static void PipeErrorSignalHandler(int sig_num)
+{
+    /* re-set the signal handler again to catch_int, for next time */
+    signal(SIGPIPE, PipeErrorSignalHandler);
+}
+
 void
 SimpleClientImpl::GlobalInitialize()
 {
     // Add intialization tasks here
+    signal(SIGPIPE, PipeErrorSignalHandler);
```

Comment:
>signal [](http://example.com/codeflow?start=4&length=6)

Does this assert?

Reply:
Yes. iOS sends SIGPIPE signal when Party tried accessing network after enter background mode.